src/geocat/comp/crop.py

```diff
@@ -2,15 +2,20 @@
 import typing
 import warnings
 import xarray as xr
-from .comp_util import _is_duck_array
+
+import geocat.comp.meteorology as gcm
 
 
 def max_daylight(
     jday: typing.Union[np.ndarray, xr.DataArray, list,
                        float], lat: typing.Union[np.ndarray, xr.DataArray, list,
                                                  float]
 ) -> typing.Union[np.ndarray, xr.DataArray, float]:
-    """Computes maximum number of daylight hours as described in the Food and
+    r""".. deprecated:: 2022.10.0 The ``crop`` module is deprecated.
+        ``max_daylight`` has been moved to the ``meteorology`` module for
+        future use.
```

Comment:
To make it super clear, should we include the correct import method in the deprecation warning? I think I might be confused if I got that warning in another package. Something like 

```suggestion
    r""".. deprecated:: 2022.10.0 The ``crop`` module is deprecated.
        ``max_daylight`` has been moved to the ``meteorology`` module for
        future use. Use ``geocat.comp.max_daylight`` or
        ``geocat.comp.meteorology.max_daylight`` for the same functionality.
```

Reply:
This is definitely more clear. I'll go through and change that for all of the deprecation warnings